.vib/wordpress/cypress/cypress/integration/wordpress_spec.js

```diff
@@ -2,88 +2,33 @@
 
 import { random } from './utils';
 
-it('contains the sample blogpost and a comment', () => {
+it('allows login/logout', () => {
+  cy.login();
+  cy.get('#login_error').should('not.exist');
+  cy.visit('wp-login.php?loggedout=true&wp_lang=en_US');
+  cy.contains('.message', 'logged out');
+});
+
+it('allows adding a comment to the sample blogpost', () => {
   cy.visit('/');
   cy.get('.wp-block-query');
   cy.get('.wp-block-post-title a').click();
-  cy.fixture('helloworld').then((hw) => {
-    cy.contains('.wp-block-post-title', hw.title);
-    cy.contains('.wp-block-post-content', hw.content);
+  cy.fixture('posts').then((post) => {
+    cy.contains('.wp-block-post-title', post.title).click();
   });
```

Comment:
This is not a hard requirement but just a thought that came to my mind. It doesn't really matter what post you comment in, so you don't really need to load a fixture here. Instead, you could simply get all the posts and click one of them.